packages/loader/container-loader/src/containerContext.ts

```diff
@@ -91,9 +91,12 @@ export class ContainerContext implements IContainerContext {
 		) => number,
 
 		/**
-		 * `unknown` should be removed once `@alpha` tag is removed from IContainerContext
+		 * `unknown` should be removed once `IContainerContext` is `@internal` or `@system`.
```

Comment:
Same feedback as https://github.com/microsoft/FluidFramework/pull/23403/files#r1926044765 - this note should move to the `@privateRemarks` block.